locator/tablets.hh

```diff
@@ -327,6 +327,13 @@ private:
     size_t _log2_tablets; // log_2(_tablets.size())
     std::unordered_map<tablet_id, tablet_transition_info> _transitions;
     resize_decision _resize_decision;
+
+    /// Returns the largest token owned by tablet_id when the tablet_count is `1 << log2_tablets`.
+    dht::token get_last_token_impl(tablet_id id, size_t log2_tablets) const;
+
+    /// Returns token_range which contains all tokens owned by the specified tablet
+    /// when the tablet_count is `1 << log2_tablets`.
+    dht::token_range get_token_range_impl(tablet_id id, size_t log2_tablets) const;
 public:
```

Comment:
I think it would be better to name them get_last_token() and get_token_range(), since they have different param set than public ones. But shouldn't block this series.

Reply:
Done.